app/src/main/java/com/jerboa/ui/components/comment/CommentNode.kt

```diff
@@ -315,12 +316,17 @@ fun LazyListScope.commentNodeItem(
                                 CommentBody(
                                     comment = commentView.comment,
                                     viewSource = viewSource,
-                                    onClick = { onCommentClick(commentView) },
+                                    onClick = {
+                                        if (!viewSource) {
+                                            onCommentClick(commentView)
+                                        }
+                                    },
                                     onLongClick = {
-                                        toggleActionBar(commentId)
+                                        if (!viewSource) {
+                                            toggleActionBar(commentId)
+                                        }
```

Comment:
What's this for? these click handlers are only for the actual text, they are not hooked on the viewSource text.

Reply:
The click events for the comment node overall are triggered when clicking and long clicking while viewing the source of a comment. This is a bit frustrating when trying to copy parts of the source out because either the toolbar will disappear or the whole comment collapses, which resets selection.

Reply:
Oh nvm they are actually bound to the `Text` remove this modifier instead 

```kt
modifier = Modifier.combinedClickable(
                    onClick = onClick,
                    onLongClick = onLongClick,
                ),
```

Reply:
Ah yeah, that makes more sense :+1: